compiler/nim.cfg

```diff
@@ -5,7 +5,6 @@ path:"llvm"
 path:"$projectPath/.."
 
 define:booting
-#import:"$projectpath/testability"
```

Comment:
I don't use the testability module often, but I don't see why it should be removed.